### PR TITLE
Docs: Remove --bypass-lock, small changes regarding compression

### PR DESCRIPTION
**Pull request by mrichtarsky**

### CODE REVIEW

File: docs/usage/help.rst.inc [hunk 1]
```diff
@@ -425,7 +425,8 @@ auto,C[,L]
     The heuristic tries with lz4 whether the data is compressible.
     For incompressible data, it will not use compression (uses "none").
     For compressible data, it uses the given C[,L] compression - with C[,L]
-    being any valid compression specifier.
+    being any valid compression specifier. This can be helpful for media files which
+    often cannot be compressed much more.
```
Comment (ThomasWaldmann) [inline docs/usage/help.rst.inc:429]: See the comment in the first line of this file: this is autogenerated, so you can't change this here.

The original text is likely somewhere in src/borg/archiver/*.py.

File: docs/quickstart.rst [hunk 2]
```diff
@@ -292,35 +292,27 @@ Backup compression
...
-
-::
+You can also use zlib and lzma instead of zstd, although zstd usually provides the
+the best compression for a given resource consumption. Please see :ref:`borg_compression`
+for all options.
 
-    $ borg create --compression lzma,N arch ~
+An interesting alternative is ``auto``, which first checks with lz4 whether a chunk is
+compressible (that check is very fast), and only if it is, compresses it with the
+specified algorithm.
```
Comment (ThomasWaldmann) [inline docs/quickstart.rst:315]: Maybe add an example for this like we have above for the other compression options?

**Comment by ThomasWaldmann** [comment]
Thanks for the PR!

**Comment by mrichtarsky** [comment]
@ThomasWaldmann This should also be ready for merge.

**Comment by ThomasWaldmann** [comment]
It would be good to backport the compression changes to 1.4-maint, can you do that please?

**Pull request merged**